auth-api/src/auth_api/resources/org.py

```diff
@@ -365,8 +365,10 @@ def delete(org_id, business_identifier):
         """Delete an affiliation between an org and an entity."""
         request_json = request.get_json()
         email_addresses = request_json.get('passcodeResetEmail')
+        should_passcode_reset = request_json.get('passcodeResetFlag')
```

Comment:
may be avoid the word flag in the variable ; call it resetPasscode or something similar to that